docs/tips/09-componentWillReceiveProps-not-triggered-after-mounting.ko-KR.md

```diff
@@ -0,0 +1,12 @@
+---
+id: componentWillReceiveProps-not-triggered-after-mounting
+title: 'componentWillReceiveProps' Mouting 후 실행되지 않습니다.
+layout: tips
+permalink: componentWillReceiveProps-not-triggered-after-mounting-ko-KR.html
+prev: controlled-input-null-value-ko-KR.html
+next: props-in-getInitialState-as-anti-pattern-ko-KR.html
+---
+
+`componentWillReceiveProps` 는 노드가 다 그려진 후 나타나지 않습니다. 이것은 의도 된 것입니다. 당신이 원하는 것은 [또다른 라이프사이클메소드 ](/react/docs/component-specs-ko-KR.html) 여기서 찾아 보실 수 있습니다.
+
+그 이유는 `componentWillReceiveProps`가 예전 속성들와 액션들의 차이점을 비교하는 로직을 종종 다루기 때문입니다. (예전 속성이 없더라도) mounting 할때 실행되지 되지 않음으로써 메소드가 어떠한지 정의하는데 도움을 줍니다. 
```

Comment:
이러면 어떨까요?
>  The reason for that is because `componentWillReceiveProps` often handles the logic of comparing with the old props and acting upon changes; not triggering it at mounting (where there are no old props) helps in defining what the method does.
>  저렇게 하는 이유는 `componentWillReceiveProps`에 종종 예전 props와 액션의 차이를 비교하는 로직이 들어가기 때문입니다. 마운트할 때 트리거되지 않으면, (예전 props가 없다고 해도) 메소드의 형태를 구별하는 데 도움이 됩니다.